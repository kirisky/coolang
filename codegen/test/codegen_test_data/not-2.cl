class Main inherits IO {
  main():Object {
    let
      t:Bool <- true,
      f:Bool <- false
    in {
      if (not not t) then out_string("not not true is true\n") else 0 fi;
      if (not f) then out_string("not false is true\n") else 0 fi;
    }
  };
};

