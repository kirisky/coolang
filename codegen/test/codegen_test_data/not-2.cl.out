not not true is true
not false is true
